src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -481,7 +481,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End If
             Next
 
-            Return If(result, VisualBasicParseOptions.Default.LanguageVersion)
+            Return If(result, VisualBasicParseOptions.Default.LanguageVersion.MapSpecifiedToEffectiveVersion)
```

Comment:
Or isthe code supposed to be ```LanguageVersion.Default.MapSpecifiedToEffectiveVersion```?

---
In reply to: [74471523](https://github.com/dotnet/roslyn/pull/12969#discussion_r74471523) [](ancestors = 74471523)